packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -145,10 +145,11 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
                       language="typescript"
                     />
                   ) : (
-                    <div>Loading...</div>
+                    <div style={{ height: '500px', display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+                      <Spinner>size={SpinnerSize.large}</Spinner>
```

Comment:
This will render a spinner with the literal text `size={SpinnerSize.large}` next to it. Should be like this instead...
```suggestion
                      <Spinner size={SpinnerSize.large}>Loading editor...</Spinner>
```

Reply:
Something must've changed before I pushed, thought I had it as a prop. You're totally right.